Edward-K1/MyDiaryWithReact
# #162235885 create react app

#### What does this PR do?
It creates an empty react and 
#### Description of Task to be completed?
Create react app using create react app
#### How should this be manually tested?

- Check into the branch 
- Run `yarn install` to install dependencies then `yarn start` to start the server.
- Navigate to localhost:3000 in a browser to confirm that the project runs successfully
